websites/Y/YNOproject/presence.ts

```diff
@@ -0,0 +1,38 @@
+const presence = new Presence({
+		clientId: "1028080411772977212",
+	}),
+	gamesWithIcons = [
+		"Uneven Dream",
+		"Amillusion",
+		"Deep Dreams",
+		"Yume Nikki",
+		"Yume 2kki",
+		".flow",
+		"Someday",
+		"Answered Prayers",
+		"Braingirl",
+	];
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "yno-logo",
+		},
+		titleInfo = document.title.split(" - "),
+		location = document.querySelector("#locationText");
+
+	if (titleInfo.length > 1 && titleInfo[1] === "YNOproject" ? true : false) {
+		presenceData.details = `Dreaming on ${titleInfo[0]}`;
+		if (gamesWithIcons.includes(titleInfo[0])) {
+			presenceData.smallImageKey = `${titleInfo[0]
+				.toLowerCase()
+				.replace(" ", "")
+				.replace(".", "")}-icon`;
+			presenceData.smallImageText = titleInfo[0];
+		}
```

Comment:
This is simpler code that also works if they add new games. Suggestion part 1/2
```suggestion
			presenceData.smallImageKey = `https://${document.location.hostname}/images/logo_${document.location.href.split('/')[3]}.png`
			presenceData.smallImageText = titleInfo[0];
```

Reply:
This makes the logo stretched, as it is not a square logo.
Is there some way to make it fit? I'm doing it manually right now because the favicon for the games is a blob created from a sprite map.
![image](https://user-images.githubusercontent.com/2998216/194721037-c79b9eca-79bf-4e6d-800f-48b4d0b16786.png)

Reply:
Replacing the logo with the door icon has a satisfactory result. Will accept your suggestion and add this change.
Thanks!
![image](https://user-images.githubusercontent.com/2998216/194726189-0a9c4c33-f0d4-4ffc-96c3-84be644b10b6.png)